assets/now.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
this thing's at https://jason.now.sh. i'm on a free account so it'll sleep; happy to submit another PR moving this to a lambda if you want to give me access to our AWS account.